src/main.rs

```diff
@@ -160,12 +153,42 @@ fn start_program(sess: &mut Session, elf: &Elf) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+/// Set rtt to blocking mode
+fn set_rtt_to_blocking(
+    core: &mut Core,
+    main_fn_address: u32,
+    rtt_buffer_address: u32,
+) -> anyhow::Result<()> {
+    // set and wait for a hardware breakpoint at the beginning of `fn main()`
+    core.set_hw_breakpoint(main_fn_address)?;
+    core.run()?;
+    core.wait_for_core_halted(Duration::from_secs(5))?;
+
+    // calculate address of rtt control block
```

Comment:
I assumed that the word we are reading is the rtt control block and documented that in the comment. Is that correct?

Reply:
Is it better like it is now?